README.rst

```diff
@@ -110,5 +110,70 @@ Here are several project that rely on MRI-NUFFT:
 - `snake-fmri <https://github.com/paquiteau/snake-fmri>`_
 - `deepinv <https://github.com/deepinv/deepinv>`_
 
+Add yours by opening a PR or an issue,  let us know how you use MRI-nufft ! 
+
+
+How to cite MRI-NUFFT
+---------------------
+
+We published MRI-NUFFT at `JOSS <https://doi.org/10.21105/joss.07743>`__ :: 
+
+    Comby et al., (2025). MRI-NUFFT: Doing non-Cartesian MRI has never been easier. Journal of Open Source Software, 10(108), 7743, https://doi.org/10.21105/joss.07743
+
+.. code:: bibtex
+          
+    @article{Comby2025, doi = {10.21105/joss.07743},
+        author = {Comby, Pierre-Antoine and Daval-Frérot, Guillaume and Pan, Caini and Tanabene, Asma and Oudjman, Léna and Cencini, Matteo and Ciuciu, Philippe and Gr, Chaithya},
```

Comment:
```suggestion
        author = {Comby, Pierre-Antoine and Daval-Frérot, Guillaume and Pan, Caini and Tanabene, Asma and Oudjman, Léna and Cencini, Matteo and Ciuciu, Philippe and GR, Chaithya},
```

Not that important, but hey I need to give some feedback :P

Reply:
We did not move from rst to md, it has always been rst but the `pyproject.toml` was referring to a non-existing `README.md`